index.html

```diff
@@ -2664,11 +2730,15 @@ <h2>
                 filters: [
                   {
                     vendorId: 0xabcd,
-                    productId: 0x1234,
                     usagePage: 0x000c,
                     usage: 0x0001,
                   },
                 ],
+                excludes: [
+                  {
+                    productId: 0x1234,
```

Comment:
I feel we should make it an error to specify a `HIDDeviceExclude` with `productId` but no `vendorId`. In this scenario we're already filtering on the vendor ID so we'll only exclude devices from that vendor, but in the general case you could filter on usages or multiple vendor IDs and then exclude based on product ID which would somewhat randomly exclude devices based on how each vendor assigns their product IDs.

Reply:
Got it.